lib/authz/plugins/resources.go

```diff
@@ -0,0 +1,103 @@
+package plugins
+
+import (
+	"fmt"
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/open-policy-agent/opa/types"
+)
+
+type Role string
+
+var (
+	RoleUser  = Role("user")
+	RoleAdmin = Role("admin")
+)
+
+type DummyUser struct {
```

Comment:
nit: rather than Dummy, name it Simple? or ... not sure?

What does this service do compared to others? comment?